src/tests/integration/mapping.test.ts

```diff
@@ -162,4 +162,48 @@ describe('/api', () => {
     const mappingData = await getMapping.json()
     expect(Object.keys(mappingData).length).toEqual(0)
   })
+
+  it('checks same subdomain can be used for different domains', async () => {
+    const subDomain = `testing${uuidv4()}`
+    const domain = 'VinDiesel'
+    const secondDomain = 'PaulWalker'
```

Comment:
declare variables close to where it gets used. In this case, `secondDomain` should be moved a few lines down, in the second `await mappingAdapter`